tests/std/tests/P0896R4_ranges_algorithm_machinery/test.cpp

```diff
@@ -533,3 +533,91 @@ namespace result_test {
     STATIC_ASSERT(test_convertible_3<in_in_out_result>());
     STATIC_ASSERT(test_convertible_3<in_out_out_result>());
 } // namespace result_test
+
+namespace permutable_test {
+    using std::forward_iterator, std::indirectly_movable_storable, std::indirectly_swappable, std::permutable;
+
+    template <int I>

```

Comment:
Nitpick: using an enum class with appropriate alternatives would greatly improve the clarity of the code.

That said if this blocks other work it can definitely wait for some rainy day

Reply:
Filed #928 to address this for all the concept "archetype" tests.